cra-tf-validate-ignore-rules.json

```diff
@@ -1,3 +1,10 @@
 {
-    "scc_rules": []
+    "scc_rules": [
+        {
+            "scc_rule_id": "rule-8cbd597c-7471-42bd-9c88-36b2696456e9",
+            "description": "Check whether Cloud Object Storage network access is restricted to a specific IP range",
+            "ignore_reason": "This module supports restricting network access using Context Based Restrictions (CBRs), however SCC does not yet support scanning for CBR rules, hence the rule currently fails. SCC CBR support is being tracked in AHA SCC-961",
```

Comment:
This is not true anymore - SCC supports this now. They no longer checks this with CRA so I think you can remove this. (PS: CRA is being replaced by a shift left SCC tool - so not too worried about CRA anymore - it provides little value)